geniza/corpus/models.py

```diff
@@ -1,29 +1,41 @@
 from django.db import models
 
 
-class Library(models.Model):
-    '''Library or archive that holds Geniza fragments'''
-    name = models.CharField(max_length=255, unique=True)
+class Collection(models.Model):
+    '''Collection at a library that holds Geniza fragments'''
+    library = models.CharField(max_length=255)
     abbrev = models.CharField('Abbreviation', max_length=255, unique=True)
-    url = models.URLField('URL', blank=True)
+    collection = models.CharField(
+        max_length=255, blank=True,
+        help_text='Collection name, if different than Library')
```

Comment:
`Collection.collection` feels weird to me, but I'm guessing there's a reason it's not `Collection.name`?

Reply:
That's a good point; but name also seems a little weird, since this field is optional